### ISSUE TITLE
A11y: Self-collapsing mode - Elements must have labels

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5039]
Ensure every form element has a label [https://dequeuniversity.com/rules/axe/4.10/label?application=AxeChrome|https://dequeuniversity.com/rules/axe/4.10/label?application=AxeChrome|smart-link] 

h3. To solve this problem, you need to fix at least (1) of the following:

* Form element does not have an implicit (wrapped) <label>
* Form element does not have an explicit <label>
* aria-label attribute does not exist or is empty
* aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
* Element has no title attribute
* Element has an empty placeholder attribute
* Element's default semantics were not overridden with role="none" or role="presentation"

!Screen Shot 2024-10-09 at 09.00.59 AM.png|width=912,height=570,alt="Screen Shot 2024-10-09 at 09.00.59 AM.png"!

!Screen Shot 2024-10-09 at 09.02.13 AM.png|width=908,height=566,alt="Screen Shot 2024-10-09 at 09.02.13 AM.png"!

!Screen Shot 2024-10-09 at 09.03.51 AM.png|width=896,height=598,alt="Screen Shot 2024-10-09 at 09.03.51 AM.png"!

!Screen Shot 2024-10-09 at 09.04.35 AM.png|width=906,height=702,alt="Screen Shot 2024-10-09 at 09.04.35 AM.png"!

!Screen Shot 2024-10-09 at 09.05.01 AM.png|width=902,height=636,alt="Screen Shot 2024-10-09 at 09.05.01 AM.png"!

!Screen Shot 2024-10-09 at 10.38.17 AM.png|width=1070,height=506,alt="Screen Shot 2024-10-09 at 10.38.17 AM.png"!

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **CUS-733**



[created from CUS-733, comment 501894]

**Comment by shopwareBot** [comment]
[public]

 > Status Change: **In Progress**

 Work on this issue has started. It is now being implemented.

 

[created from CUS-733, comment 511045]

**Comment by shopwareBot** [comment]
[public] 

 > Status Change: **Open**

 This issue is ready for implementation.

 

[created from CUS-733, comment 511046]

**Comment by shopwareBot** [comment]
[public]

 > Status Change: **In Review**

 Work on this ticket is complete and is now pending review.



[created from CUS-733, comment 511510]
